doc/src/guide/usage.asciidoc

```diff
@@ -136,12 +150,12 @@ and the worker will not be checked in when the original process dies.
 
 The process calling this function must be the current owner of the worker.
 
-The process receiving the worker is sent a message `{'HNC-WORKER-TRANSFER', Worker, FromPid, GiftData}`.
+The process receiving the worker is sent a message `{'HNC-WORKER-TRANSFER', WorkerRef, FromPid, GiftData}`.
 
 [source,erlang]
 ----
-Result = hnc:give_away(PoolName, Worker, OtherProcess, GiftData).
-Result = hnc:give_away(PoolName, Worker, OtherProcess, GiftData, Timeout).
+Result = hnc:give_away(WorkerRef, OtherProcess, GiftData).
+Result = hnc:give_away(WorkerRef, OtherProcess, GiftData, Timeout).
```

Comment:
The argument description below still describes `PoolName` and `Worker` instead of `WorkerRef`.

Reply:
Done.